src/versionManager.ts

```diff
@@ -50,6 +52,21 @@ export interface Config {
 
 /** Returns the path to the executable */
 export async function install(config: Config, version: semver.SemVer): Promise<string> {
+    const key = config.exeName + version.raw;
+    const entry = inProgressInstalls.get(key);
+    if (entry) {
+        return await entry;
+    }
+
+    const fn = installGuarded.apply(null, [config, version]);
```

Comment:
Is `apply` really needed here?
```suggestion
    const fn = installGuarded(config, version);
```

Reply:
It probably isn't.